### ISSUE TITLE
Fatal error when trying to get messages without running validation

### ISSUE THREAD

**Issue by skoop** [particle-php/Validator#69]
I get a fatal error when running the following code:

```php
$validator = new Particle\Validator\Validator();

echo $validator->getMessages();
```

While the above code logically incorrect (no validation has been run), it could happen and should at least not fatal.

**Comment by RickvdStaaij** [comment]
This is pretty much solved in version 2 aka the master brach, as you can't have getMessagea without the ValidationResult. A quick fix for version one might come in handy though

**Comment by skoop** [comment]
#70 contains a fix for this. let me know if this is acceptable

**Comment by RickvdStaaij** [comment]
The issue was resolved in #70 and released in `v1.0.5`, closing issue.
